package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "kvue",
-	"version": "1.0.0",
+	"version": "2.10.0",
 	"lockfileVersion": 2,
 	"requires": true,
 	"packages": {
 		"": {
 			"name": "kvue",
-			"version": "1.0.0",
+			"version": "2.10.0",
```

Comment:
@emuvente Guessing this is fine and will now get updated in the build/deploy process